pkg/cmd/operator/analyze.go

```diff
@@ -0,0 +1,85 @@
+package operator
+
+import(
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/scylladb/scylla-operator/pkg/genericclioptions"
+	"k8s.io/apimachinery/pkg/util/errors"
+)
+
+type AnalyzeOptions struct{
+	Mode string
+	GatherDirPath string
+}
+
+func (o *AnalyzeOptions) Validate() error {
+	var errs []error
+
+	if len(o.Mode) != 0 && o.Mode != "live" && o.Mode != "gathered"{
+		errs = append(errs, fmt.Errorf("Invalid mode. Use either 'live' or 'gathered'" ))
+	}
+
+	if o.Mode == "gathered" && len(o.GatherDirPath) == 0 {
+		errs = append(errs, fmt.Errorf("dir-path not specified"))
+	}
+
+	if len(o.Mode) == 0 && len(o.GatherDirPath) == 0 {
+		errs = append(errs, fmt.Errorf("Set flag use=live or specify directory path for gathered data"))
+	}
+
+	return errors.NewAggregate(errs)
+}
+
+func (o *AnalyzeOptions) Complete() error {
+	return nil
+}
+
+func (o *AnalyzeOptions) Run(streams genericclioptions.IOStreams, cmd *cobra.Command) error {
+	fmt.Printf("Analyze run with options: mode='%s', gatherDirPath='%s'\n", o.Mode, o.GatherDirPath)
+
+	// TODO
```

Comment:
Lets not leave empty TODOs, it should have description what is to do. If there's a github issue, link should be there.